src/components/UpcomingShiftCard/UpcomingShiftCard.jsx

```diff
@@ -3,13 +3,40 @@ import { Card } from 'react-bootstrap';
 import { FaChevronRight } from 'react-icons/fa';
 import './UpcomingShiftCard.css';
 
-export default () => (
+function styleDate(data) {
+  const date = data.split(' ')[0];
+  const year = date.split('-')[0];
+  let month = date.split('-')[1];
+  const day = date.split('-')[2];
+  const months = [null, 'January', 'February', 'March', 'April', 'May', 'June', 'July', 'August', 'September', 'October', 'November', 'December'];
+
+  month = months[parseInt(month, 10)];
+
+  return (`${month} ${day}, ${year}`);
+}
+
+function styleTime(data) {
+  const time = data.split(' ')[1];
+  const hours = time.split(':')[0];
+  const min = time.split(':')[1];
+  return (`${hours}:${min}`);
+}
+
+export default (cardData) => (
 
   <Card>
     <Card.Body>
       <div>
-        <Card.Title>Tuesday, March 9th</Card.Title>
-        <Card.Text>8:00 PM - 8:00 AM</Card.Text>
+        <Card.Title>
+          {console.log(cardData)}
+          {styleDate(cardData.cardData.from)}
+        </Card.Title>
+        <Card.Text>
+          {styleTime(cardData.cardData.from)}
+          AM-
+          {styleTime(cardData.cardData.to)}
```

Comment:
Add a space after AM to match mockups
```suggestion
          {styleTime(cardData.cardData.from)}
          AM -
          {styleTime(cardData.cardData.to)}
```